### PR TITLE
qcontainer:add throttle support for cdrom device

### PR DESCRIPTION
**Pull request by qingwangrh**
ID:1920866
Signed-off-by: qingwangrh <qinwang@redhat.com>

### CODE REVIEW

**Comment by qingwangrh** [comment]
@YongxueHong @zhencliu Please help to review,thanks.

File: virttest/qemu_devices/qcontainer.py [hunk 1]
```diff
@@ -2992,11 +2993,10 @@ def fs_define_by_params(self, name, params, bus=None):
         if driver == "virtio-fs":
             binary = params.get("fs_binary", "/usr/libexec/virtiofsd")
             extra_options = params.get("fs_binary_extra_options")
-            enable_debug_mode = params.get('fs_enable_debug_mode', 'no') == "yes"
             sock_path = os.path.join(data_dir.get_tmp_dir(),
                                      '-'.join((self.vmname, name, 'virtiofsd.sock')))
             vfsd = qdevices.QVirtioFSDev(name, binary, sock_path,
-                                         source, extra_options, enable_debug_mode)
```
Comment (YongxueHong) [inline virttest/qemu_devices/qcontainer.py:2999]: Would you explain to me the reason for this part?
Thanks.

File: virttest/qemu_devices/qcontainer.py [hunk 2]
```diff
@@ -145,7 +145,7 @@ def get_qmp_cmds(qemu_binary, workaround_qemu_qmp_crash=False):
                              shell=True,
                              verbose=False)
         # Some architectures (arm) require machine type to be always set
-        failed_pattern = r'(?:kvm_init_vcpu.*failed)|(?:machine specified)'
+        failed_pattern = r'(?:kvm_init_vcpu failed)|(?:machine specified)'
```
Comment (YongxueHong) [inline virttest/qemu_devices/qcontainer.py:148]: Would you explain to me the reason for this part?
Thanks.

**Review by YongxueHong — APPROVED** [review]
LGTM

**Comment by zhencliu** [comment]
LGTM

**Pull request merged**